### DISCUSSION TITLE
Fill Dropdown List dependent of another selection

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#393] [Q&A] [answered]
Hi

In a document collection we have documents (e.g. shops), where we have the fields country and city. Both reference a collection.
Is it possible in the create/editform to have two listboxes, where the 2nd one (city) shows values filtered by the first one?
Example
Listbox 1 Country: <country> 
Listbox 2 City: <shows only cities of the selected country in listbox 1>

**Comment by fgatti675** [comment, marked as answer]
Hi @Andre3424 
You can achieve this with conditional fields.
You would have one regular property `country` as a reference, and a conditional property `city`
You can filter the values of city from the value of country
https://firecms.co/docs/properties/conditional_fields

**Reply by Amtech281** [reply]
@fgatti675 thanks! thats great
